build/getDependencies-windows.sh

```diff
@@ -132,6 +132,7 @@ mkdir -p ../DistFiles/pdf
 mkdir -p ../Downloads
 mkdir -p ../MercurialExtensions
 mkdir -p ../MercurialExtensions/fixutf8
+mkdir -p ../build
 mkdir -p ../build/
```

Comment:
What's the difference between these two commands? I find it surprising that we need both.